### PR TITLE
[WIP] first draft

### PR DESCRIPTION
**Pull request by arbll**

### CODE REVIEW

File: docs-content/index.html.md.erb [hunk 1]
```diff
@@ -1,115 +1,99 @@
...
     </tr>
     <tr>
         <td>Software component version</td>
-        <td>vYOUR-SOFTWARE-COMPONENT-VERSION-NUMBER</td>
+        <td>v6.1.3</td>
     </tr>
     <tr>
         <td>Compatible Ops Manager version(s)</td>
-        <td>vYOUR-COMPATIBLE-OPS-MAN-VERSION (Ex. v1.11.x, v1.12.x, and v2.0.x)</td>
+        <td>TODO</td>
```
Comment (dsahni) [inline docs-content/index.html.md.erb]: Would this be the same as the earlier tile, i.e. Ops Manager version `v2.0.x`?

File: docs-content/installing.html.md.erb [hunk 2]
```diff
@@ -1,28 +1,23 @@
...
 ---
 
-This topic describes how to install and configure YOUR-PRODUCT-NAME.
+This topic describes how to install and configure Datadog Application Monitoring for PCF.
 
-##<a id='install'></a> Install and Configure YOUR-PRODUCT-NAME
+##<a id='install'></a> Install and Configure Datadog Application Monitoring for PCF.
 
-1. Download the product file from Pivotal Network.
+1. Download the product file for the **Datadog Application Monitoring for PCF** from Pivotal Network.
```
Comment (dsahni) [inline docs-content/installing.html.md.erb:10]: Would the BOSH Health Monitor also be required as the first step here? (similar to our first tile: https://docs.pivotal.io/partners/datadog/installing.html)

File: docs-content/index.html.md.erb (hunk 1 above)
Comment (gmmeyer) [reply docs-content/index.html.md.erb]: I think we could fully support earlier versions, and we have customers, I know, on earlier versions.

File: docs-content/index.html.md.erb [hunk 3]
```diff
@@ -1,115 +1,99 @@
...
     </tr>
     <tr>
-        <td>Compatible Pivotal Application Service (formerly known as Elastic Runtime) version(s)</td>
-        <td>vYOUR-COMPATIBLE-PIVOTAL-APPLICATION-SERVICE-VERSION (Ex. v1.11.x, v1.12.x, and v2.0.X)</td>
+        <td>Compatible Elastic Runtime version(s)</td>
+        <td>TODO</td>
```
Comment (gmmeyer) [inline docs-content/index.html.md.erb]: This should just be the same as the opsman versions

File: docs-content/release-notes.html.md.erb [hunk 4]
```diff
@@ -1,58 +1,14 @@
...
 ---
 
-**This topic should include the Release Notes for your product. For examples, see the 
-[GCP Service Broker Release Notes](http://docs.pivotal.io/gcp-sb/release-notes.html) or the 
-[ISS Knowtify Search Analytics Release Notes](http://docs.pivotal.io/knowtify/release.html).
-When you release a new version, add new release notes to this file, keeping the older notes below.**
+These are release notes for Datadog Application Monitoring for Pivotal Cloud Foundry (PCF).
 
-These are release notes for YOUR-PRODUCT-NAME.
+##<a id="ver"></a> v0.9.0
```
Comment (gmmeyer) [inline docs-content/release-notes.html.md.erb]: Let's call it v1.0.0! I only made it that version as a testing release

File: docs-content/using.html.md.erb [hunk 5]
```diff
@@ -1,15 +1,14 @@
...
-title: Using YOUR-PRODUCT-NAME
+title: Using Datadog Application Monitoring for PCF
 owner: Partners
 ---
 
-This topic describes how to use YOUR-PRODUCT-NAME.
+This topic describes how to use Datadog Application Monitoring for PCF.
 
-##<a id='using'></a> Using YOUR-PRODUCT-NAME
+##<a id="view"></a> Monitor your application performance
```
Comment (gmmeyer) [inline docs-content/using.html.md.erb:8]: While I'm good with us mostly just linking to their docs, we don't have to provide a full tutorial here, I think we should give a little more context and explanation for what the buildpack does. 

However, I also think we need to add a tutorial here as to how to install the buildpack. The one I have on the buildpack readme will probably work pretty well.

**Pull request merged**